app/models/space.rb

```diff
@@ -39,10 +40,24 @@ def metrics
     end
   end
 
+  def guesstimates_of_type(types)
+    return [] if graph.nil?
+
+    return graph['guesstimates'].select {|guesstimate| types.include? guesstimate["guesstimateType"]}
+  end
+
   def is_public?
     !self.is_private
   end
 
+  def is_searchable?
+    is_public? and
+    has_real_name? and
+    guesstimates_of_type(["UNIFORM", "NORMAL"]).length > 0 and
+    guesstimates_of_type(["FUNCTION"]).length > 0 and
+    metrics.length > 3
```

Comment:
i'd probably rename the top three into a separate method, as ``is_interesting?`` or similar.  It could be private now.